package.json

```diff
@@ -7,6 +7,9 @@
     "type": "git",
     "url": "https://github.com/daostack/subgraph.git"
   },
+  "engines": {
+    "node": "8"
```

Comment:
should be 10.30.0 (recommended)

Reply:
Also better use `>=`

Reply:
https://github.com/daostack/subgraph/pull/6/commits/be75e747f40475b2e3d8cdbba52fa4526f504742

Reply:
should be 10.13.0

Reply:
why does the minor version matter? just `10` is enough imo.